lib/config/types.ts

```diff
@@ -11,6 +11,7 @@ export type RenovateConfigStage =
   | 'pr';
 
 export type RepositoryCacheConfig = 'disabled' | 'enabled' | 'reset';
+export type RepositoryCacheType = 'local'; // add "s3" and "redis" once implemented
```

Comment:
same

Reply:
same

Reply:
lets add string now, even it it's no used yet, makes testing not supported implementations easier